packages/vuetify/src/directives/resize/index.ts

```diff
@@ -1,13 +1,15 @@
-import { VNodeDirective } from 'vue/types/vnode'
+import { DirectiveBinding, ObjectDirective } from 'vue'
 
-interface ResizeVNodeDirective extends VNodeDirective {
-  value?: () => void
-  options?: boolean | AddEventListenerOptions
+interface ResizeDirectiveBinding extends DirectiveBinding {
+  value: (() => void) | undefined
 }
 
-function inserted (el: HTMLElement, binding: ResizeVNodeDirective) {
+function mounted (el: HTMLElement, binding: ResizeDirectiveBinding) {
   const callback = binding.value!
-  const options = binding.options || { passive: true }
+  const options = {
```

Comment:
how about (ts.next version)
```ts
const options = binding.modifiers?.active 
  ? (binding.options ?? false)
  : passiveEventOptions(binding.options ?? false)
```

Reply:
maybe. do we have a version that supports it?

Reply:
Not yet, although you can skip the ts.next part for now and apply the rest of the proposal

```ts
const options = (binding.modifiers && binding.modifiers.active) 
  ? (binding.options || false)
  : passiveEventOptions(binding.options || false)
```

Reply:
sure! could you create a suggestion please?

Reply:
We aren't supporting IE so PASSIVE_SUPPORTED can be assumed to be true